### DISCUSSION TITLE
Segments surface property

### DISCUSSION THREAD

**Discussion by BruceHarold** [OvertureMaps/data#126] [Discussion]
Hi, I see two forms of surface definition when there is no geometric scoping, e.g.:

{"class":"primary","surface":"paved"}
{"class":"primary","surface":[{"value":"paved"}]}

Do they mean the same thing, paved from 0.0-1.0 scope?

Thanks.

**Comment by brad-richardson** [comment]
This was likely a bug in the old JSON-formatted road column and should no longer be an issue
